modules/python/python-value-pairs.c

```diff
@@ -36,16 +36,16 @@ python_worker_vp_add_one(const gchar *name,
 {
   const LogTemplateOptions *template_options = (const LogTemplateOptions *)((gpointer *)user_data)[0];
   PyObject *dict = (PyObject *)((gpointer *)user_data)[1];
-  gboolean fallback = template_options->on_error & ON_ERROR_FALLBACK_TO_STRING;
 
   PyObject *obj = py_obj_from_log_msg_value(value, value_len, type);
-  if (!obj && fallback)
-    {
-      obj = py_obj_from_log_msg_value(value, value_len, LM_VT_STRING);
-    }
```

Comment:
I've found this in an error message improvement commit.
It is probably here because of the `type_cast_error` fallback.

Reply:
in the meantime that _log_py_object_conversion_error_with_invalid_type() was removed from master and I didn't fix it correctly during rebase.

I am running a compile test on each patch in the series, hopefully that resolves it.